x-pack/plugins/security_solution/public/detection_engine/rule_creation/components/alert_suppression_edit/fields.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const ALERT_SUPPRESSION_FIELDS = 'alertSuppressionFields' as const;
```

Comment:
Ideally fields should relay its purpose by name. In the same time `groupByFields` was quite generic making it hard to comprehend it's related to alert suppression.

In fact form fields could be anything since we transform form field values to rule fields. Exporting field name constants helps avoiding using hardcoded field names and simplify Alert Suppression component reusability.